source/features/prevent-pr-merge-panel-opening.tsx

```diff
@@ -0,0 +1,36 @@
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+let controller: AbortController;
+
+async function sessionResumeHandler(): Promise<void> {
+	await Promise.resolve(); // The `session:resume` event fires a bit too early
+	const cancelMergeButton = select('.merge-branch-form .js-details-target');
+	if (cancelMergeButton) {
+		cancelMergeButton.click();
+		controller.abort();
+	}
+}
+
+function init(): void {
+	controller = new AbortController();
+	document.addEventListener('session:resume', sessionResumeHandler, {signal: controller.signal});
```

Comment:
It's easier to remove the handler via removeEventHandler than having a global controller